### ISSUE TITLE
[Feature] show top ram programs

### ISSUE THREAD

**Issue by Helloyunho** [gao-sun/eul#81]
Checking top CPU programs and top network programs is pretty handy, but I think it could be better if it shows top ram programs also.

Any plans to add this?

**Comment by gao-sun** [comment]
absolutely

**Comment by jevonmao** [comment]
I would like to have a go at this. I'm new to this project, still not very familiar with the codebase yet. So I might need some help.

**Comment by jevonmao** [comment]
# Issue \# 81 show top ram programs design #

Created by Jevon Mao on 2020/1/18. 

## TL;DR ##

This is a design doc that outlines my plan, tracks my progress, and document any related information to implementing [#81](https://github.com/gao-sun/eul/issues/81) that is a section showing top ram processes similar to the existing top CPU processes.

## Resource References ##

1. https://ss64.com/osx/top.html (Mac terminal command to get top ram processes)

## Implementation Steps ##

- [ ] 1. Test out UI design by creating a SwiftUI view with mock information
- [ ] 2. Use a terminal command to get top ram processes, and parse the command within project
- [ ] 3. Create data model and related functions to store top ram processes
- [ ] 4. Connect view model to SwiftUI view
- [ ] 5. Pull request, improve code if requested by reviewer

**Comment by gao-sun** [comment]
your PR looks great @jevonmao! just left 2 comments. let me if you need any help
